### DISCUSSION TITLE
Preparations and Prerequisites - March 2021

### DISCUSSION THREAD

**Discussion by ghobona** [opengeospatial/developer-track#3] [Announcements]
Please let the participants what they need to do before the session

**Comment by ghobona** [comment]
Ping: @tomkralidis @pvgenuchten @francbartoli @ShaneMill1 @iandruska-ibl @m-burgoyne

**Comment by ghobona** [comment]
Initial set of Registered Participants and Instructors includes: @joanma747 @pvretano @bpross-52n @tomkralidis @iandruska-ibl @francbartoli @solson-nws @ShaneMill1 @chris-little @gbuehler @heidivanparys @PeterTrevelyan @cportele @YanmeiWu @BartDeLathouwer  @RobertNL @jademonkee @AndrewRadburn @ingosimonis @MarcusAlzona @prushforth @StephanSiemen @m-burgoyne @brainwaive-th @RyanAhola @thanchevici @mklopfer1 @kcski @scontarinis @NazihFino @geofizzydrink @KoalaGeo @amuhsen

**Reply by tomkralidis** [reply]
Hi all: for the practical work sessions involving [pygeoapi](https://pygeoapi.io), please see the pygeoapi [installation](https://docs.pygeoapi.io/en/latest/installation.html) docs.  The pygeoapi exercises will involve having a working instance.  Please feel free to ask for support here or in the pygeoapi mailing list or Gitter channel.

**Reply by skyNacho** [reply]
Thanks @tomkralidis Which method do you recommend: git cloning, pip, conda, docker image or all should be equally valid for the exercises?

**Reply by tomkralidis** [reply]
Hi @skyNacho , I would recommend git cloning given the exercises will involving updating configuration, process and loading metadata. Let us know how you make out (or chat with us on [Gitter](https://gitter.im/geopython/pygeoapi)).

**Reply by chris-little** [reply]
@ghobona Where should I put my PowerPoint slides on the background of the OGC API-EDR lease? They currently sit in the OGC Portal in the Meetings/Met Ocean DWG folder.

**Comment by ghobona** [comment]
New registrations: @fbehr @nilshempelmann @jgranek @jlundnielsen @jmprevost

**Comment by francbartoli** [comment]
Hi all: for the specific session about Tiles in pygeoapi, beyond the instructions given by @tomkralidis, there is the requirement to have [Docker](https://www.docker.com/products/docker-desktop) installed in order to generate vector tiles with [tippecanoe](https://github.com/mapbox/tippecanoe) and place them in a local bucket with [Minio](https://github.com/minio/minio).

Let's chat with us on [Gitter](https://gitter.im/geopython/pygeoapi) if you need more guidance

**Comment by tomkralidis** [comment]
Hi all: for the suite of pygeoapi presentations, we have put forth a presentation at https://docs.google.com/presentation/d/13A5QfSgepiOmd4ttjr376MjRKgx2zdJaty0LPXgqkQk for tomorrow.  Slides 2-7 are common to all practical sessions.

Please feel free to contact us if you have any questions or comments. Thanks!

**Comment by ghobona** [comment]
An example OpenAPI definition document that implements OGC API - Tiles and OGC API - Maps can also be found at https://app.swaggerhub.com/apis/UAB-CREAF/ogc-api-map-tiles-opf-xmp-mt-more-1-collection/1.0.0

**Comment by ghobona** [comment]
An example OpenAPI definition document that implements OGC API - Processes can also be found at https://app.swaggerhub.com/apis/geoprocessing/ogcapi-processes/1.0-draft.6-SNAPSHOT

**Comment by serich** [comment]
If anybody experiences difficulty sharing their screen via the GoToMeeting desktop client, consider joining via browser by visiting https://app.gotomeeting.com/ and entering the session ID 704254605 (for the March 24 telecon). The interface might not be as rich as the desktop, but at least the screen-sharing should work.

There was a [similar problem last year](https://community.logmein.com/t5/GoToMeeting-Discussions/Unable-to-share-screen-since-upgrading-to-Big-Sur/td-p/257159). Part of the resolution was to remove GoToMeeting completely (navigate to Applications Folder and delete the GoToMeeting application). Then [re-install via DMG (Mac)](https://support.goto.com/meeting/help/install-via-dmg-mac-g2m010014). But I'm not sure whether this link is still current.

My desktop client seems to be OK for now: Mac Big Sur 11.2.3, GoToMeeting Version 10.15.0 Build 19228.
